stage1/data/Example_i.csv

```diff
@@ -0,0 +1,7 @@
+id,name,some_capitalized,atleast_one_capitalized,first_letter_capitalized,has_suffix_salutation,start_position,distance_to_period,distance_to_closest_keyword,frequency,label

```

Comment:
I guess these csvs should be in a separate folder and keep the data folder clean.